roles/artemis/templates/artemis.env.j2

```diff
@@ -238,3 +238,25 @@ EUREKA_INSTANCE_INSTANCEID='Artemis:{{ artemis_eureka_instance_id }}'
 {% endif %}
 LOGGING_FILE_NAME='{{ artemis_working_directory }}/artemis.log'
 MANAGEMENT_PROMETHEUS_METRICS_EXPORT_ENABLED='true'
+###############################################################################
+# Helios
+###############################################################################
+{% set is_build_agent = continuous_integration.localci.is_build_agent
+                         if (continuous_integration is defined
+                             and continuous_integration.localci is defined
+                             and continuous_integration.localci.is_build_agent is defined)
+                         else false %}
+# Shown on GH → Settings → Environments (must match exactly)
+{% if helios_environment_name is defined and helios_environment_name != "" %}
+HELIOS_ENVIRONMENT_NAME="{{ helios_environment_name }}"
+    {% if helios_prod_secret_key is defined and helios_prod_secret_key != "" %}
+HELIOS_PROD_SECRET_KEY="{{ helios_prod_secret_key }}"
+    {% endif %}
+    {% if helios_staging_secret_key is defined and helios_staging_secret_key != "" %}
+HELIOS_STAGING_SECRET_KEY="{{ helios_staging_secret_key }}"
+    {% endif %}
+{% endif %}
+{#-- Flag build agents for later use inside application.yml --#}
+{% if is_build_agent %}
+IS_BUILD_AGENT=true
+{% endif %}
```

Comment:
This is significantly different from the `application-prod.yml.j2`. It should be an exact replica of values, just as a Spring-compatible ENV file instead of a Spring-compatible YML file